### PR TITLE
Allow negative voting

### PR DESCRIPTION
**Pull request by tsiegleauq**
Adds "no" as the opposite of "votes" as assignment poll method

### CODE REVIEW

**Comment by tsiegleauq** [comment]
@FinnStutzenstein most of the server part is currently missing.

esp: "n" is similar to votes, however, it sends "1" per every poll option without any other hint.

**Comment by FinnStutzenstein** [comment]
> however, it sends "1" per every poll option without any other hint.

This is fine.

I added `global_yes` completly analog to `global_no` and `global_abstain` (required by Emanuel). I also implemented the `N` mode. The payload is the same as for the votes mode: `{<option_id>: <amount>}` with amount being positive. The only differnece is, that the votes are saved into no instead of yes. Also adapted the analog voting.

Please see, if the client needs more changes now (I did some..)

**Comment by FinnStutzenstein** [comment]
@tsiegleauq I renamed the `votes` pollmethod and 100% base for assignments to `Y`

File: client/src/app/site/assignments/modules/assignment-poll/services/assignment-poll.service.ts [hunk 1]
```diff
@@ -109,7 +117,14 @@ export class AssignmentPollService extends PollService {
         const tableData: PollTableData[] = poll.options
             .sort((a, b) => {
                 if (this.sortByVote) {
-                    return b.yes - a.yes;
+                    if (poll.pollmethod === AssignmentPollMethod.N) {
+                        // most no on top:
+                        return b.no - a.no;
+                        // least no on top:
+                        // return a.no - b.no;
```
Comment (emanuelschuetze) [inline client/src/app/site/assignments/modules/assignment-poll/services/assignment-poll.service.ts:124]: We need the most 'yes' on top - for a no election. Currently, it does not work!

**Comment by tsiegleauq** [comment]
@emanuelschuetze updated

**Review by emanuelschuetze — APPROVED** [review]

**Pull request merged**